spec/validators/cat_validator_spect.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+LORE_IPSUM_250 = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam pretium felis sit amet libero congue, sed bibendum tortor eleifend. Nam imperdiet pellentesque magna, sed consectetur est egestas vitae. Praesent tempus lacus nec finibus tincidunt amet.'
+LORE_IPSUM_513 = + LORE_IPSUM_250 + LORE_IPSUM_250 + "1234567890123"
+
+RSpec.describe Cat, :type => :model do
+  context 'given a cat with only name ' do
+    it 'must be valid' do
+      valid_cat = Cat.new(name: "Kitty")
```

Comment:
Instead of creating a Cat and a Colony, you should use Rspec mocks (mock_model). Take a look at the repo files to see how it's done.

Reply:
I'll take a look this night

Reply:
Not really, I'm looking the problem right now, instead of being working on my FDP